source/exec.tex

```diff
@@ -5672,3 +5690,845 @@
+\tcode{set_error_t(E)} for some type \tcode{E}.
+
+\pnum
+The type alias \tcode{completion_signatures} is a specialization
+of \tcode{execution::completion_signatures} with the template
+arguments (in unspecified order):
+
+\begin{itemize}
+\item \tcode{set_value_t()} if \tcode{T} is \tcode{void}, and
+\tcode{set_value_t(T)} otherwise;
+\item template arguments of the specialization of
+\tcode{execution::completion_signatures} denoted by \tcode{error_types};
+and
+\item \tcode{set_stopped_t()}.
+\end{itemize}
+
+\pnum
+\tcode{allocator_type} shall meet the \oldconcept{Allocator}
+requirements\iref{allocator.requirements.general}.
+
+\rSec3[task.members]{\tcode{task} Members}
+
+\begin{itemdecl}
+task(task&& other) noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Initializes \exposid{handle} with \tcode{exchange(other.\exposid{handle},
+\{\})}.
+\end{itemdescr}
+
+\begin{itemdecl}
+~task();
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+
+\begin{codeblock}
+if (@\exposid{handle}@)
+  @\exposid{handle}@.destroy();
+\end{codeblock}
+\end{itemdescr}
+
+\begin{itemdecl}
+template<@\libconcept{receiver}@ R>
+@\exposid{state}@<R> connect(R&& recv);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+\tcode{bool(\exposid{handle})} is \tcode{true}.
+
+\pnum
+\effects
+Equivalent to: \tcode{return \exposid{state}<R>(exchange(\exposid{handle},
+\{\}), std::forward<R>(recv));}
+\end{itemdescr}
+
+\rSec3[task.state]{\tcode{Class template \tcode{task::state}}}
+
+\begin{codeblock}
+namespace std::execution {
+  template<class T, class Environment>
+    template<@\libconcept{receiver}@ Rcvr>
+  class task<T, Environment>::@\exposid{state}@ { // \expos
+  public:
+    using operation_state_concept = operation_state_t;
+
+    template<class R>
+    @\exposid{state}@(coroutine_handle<promise_type> h, R&& rr);
+    ~@\exposid{state}@();
+    void start() & noexcept;
+
+private:
+    using @\exposidnc{own-env-t}@ = @\seebelow@;     // \expos
+    coroutine_handle<promise_type> @\exposidnc{handle}@;  // \expos
+    remove_cvref_t<Rcvr>              @\exposidnc{rcvr}@;    // \expos
+    @\exposid{own-env-t}@                      @\exposidnc{own-env}@; // \expos
+    Environment                    @\exposidnc{environment}@; // \expos
+  };
+}
+\end{codeblock}
+
+\begin{itemdescr}
+\pnum
+The type \exposid{own-env-t} is \tcode{Environment::template
+env_type<decltype(get_env(\linebreak declval<Rcvr>()))>} if that % avoid Overfull
+qualified-id is valid and denotes a type, \tcode{env<>} otherwise.
+\end{itemdescr}
+
+\begin{itemdecl}
+template<class R>
+@\exposid{state}@(coroutine_handle<promise_type> h, R&& rr);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Initializes
+
+\item \exposid{handle} with \tcode{std::move(h)};
+\item \exposid{rcvr} with \tcode{std::forward<R>(rr)};
+\item \exposid{own-env}
+    with \tcode{\exposid{own-env-t}(get_env(\exposid{rcvr}))} if that expression
```

Comment:
This indentation within bullets is wrong. Just get rid of it an make it all flat.

Reply:
I have removed the leading indentation with [this commit](https://github.com/cplusplus/draft/pull/8032/commits/2ec36bb49f287d73cd17e8dcf9ebbb0f13d3ceed).